fancy-explosions/BangazonSite
# Js issue9 and 10

# Description


Fixes Issue #9 
Fixes Issue #10 

## Type of change

Please delete options that are not relevant.



# Testing Instructions

git fetch --all
git checkout js-issue9
run it
log in
in the nav bar search a product should should in the list after hit enter
then click where it say email address then select payment options
then delete 1 of your payment options



# Checklist:
